cg/meta/upload/gt.py

```diff
@@ -126,3 +126,17 @@ def _is_variant_file(genotype_file: housekeeper_models.File):
 
     def _get_genotype_files(self, version_id: int) -> list:
         return self.hk.files(version=version_id, tags=["genotype"]).all()
+
+    @staticmethod
+    def genotype_check(case_obj: models.Family) -> bool:
+        """Check if balsamic case is contains WGS and normal sample"""
+
+        samples = [link.sample for link in case_obj.links]
+        for sample in samples:
+            # check if normal sample
+            if not sample.is_tumour:
+                # Check if WGS sample
+                if "wgs" == sample.application_version.application.prep_category:
+                    return True
+
+        return False
```

Comment:
Skulle det här funka?
Prior:
https://github.com/Clinical-Genomics/cg/pull/1547/files#diff-b8c6910e609c186a438ff896af8edcce5ac06dfcfd05029e322b287e1b9624e9
sample_prep_category: str = sample.application_version.application.prep_category

```suggestion
           return True if not sample.is_tumour and PrepCategory.WHOLE_GENOME_SEQUENCING == sample_prep_category else False

```

Reply:
I agree that we should use a constant instead of hard-coding it, so I will implement that!

However, checking genotype compatibility with one logical check will be difficult since there might be balsamic cases where the tumor sample is at index 0 in the sample list. If so, it will return false even though there might be a normal case at index 1 that should be uploaded to Genotype.

Reply:
👍 ! Good point! Could you just remove the comments? I think the code is clear enough

Reply:
Yes, I will do that!